src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -106,6 +115,10 @@ def main(phabricator,
                     error=e,
                 )
 
+    # Publish full runtime
+    end_time = time.time()
+    stats.api.timing('analysis.runtime', end_time - start_time)
```

Comment:
It would be nice to split this up, so we can monitor how long the various steps of the analysis are taking.